app/views/govuk_component/docs/tasklist_header.yml

```diff
@@ -0,0 +1,33 @@
+name: Tasklist header
+description: Shows that a content page is part of a tasklist.
+body: |
+  The component indicates to the user that the current page is part of a tasklist, and can provide a link to that tasklist.
+accessibility_criteria: |
+  The component is designed to go into the top of an existing content page and should not interfere with the heading structure of the page, so therefore should not contain a heading tag.
+
+  The component must:
+
+  - provide a skip link to the tasklist on the page, if that tasklist is present
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      title: 'Learn to drive a car: step by step'
+  with_a_link:
+    data:
+      title: 'Having children: step by step'
+      path: /notalink
+  with_a_skip_link:
```

Comment:
Why is this aimed more at keyboard users only? Wouldn't all users benefit from being able to click to go to the tasklist? (I may just be missing something as I can't see the tasklist header in context!)

Reply:
The skiplink is invisible unless it receives focus, which can only be achieved by using a keyboard or a screenreader. I'll update the yml to explain this.

Here's the prototype where you can see the tasklist header in context (although it doesn't have a skiplink) - https://govuk-services-prototype.herokuapp.com/apply-first-provisional-driving-licence